doc/how-to/getting_started_net_box.rst

```diff
@@ -0,0 +1,155 @@
+.. _getting_started_net_box:
+
+Getting started with net.box
+============================
+
+The tutorial shows how to work with some common ``net.box`` methods.
+
+For more information about the ``net.box`` module,
+check the :ref:`corresponding module reference <net_box-module>`.
+
+Sandbox configuration
+---------------------
+
+The sandbox configuration for the tutorial assumes that:
+
+*   The Tarantool instance is running on ``localhost 127.0.0.1:3301``.
+*   There is a space named ``tester`` with a numeric primary key.
+*   The space contains a tuple with the key value = ``800``.
+*   The current user has read, write, and execute privileges.
+
+Use the commands below for a quick sandbox setup:
+
+..  code-block:: lua
+
+    box.cfg{listen = 3301}
+    s = box.schema.space.create('tester')
+    s:create_index('primary', {type = 'hash', parts = {1, 'unsigned'}})
+    t = s:insert({800, 'TEST'})
+    box.schema.user.grant('guest', 'read,write,execute', 'universe')
+
+Creating a net.box connection
+-----------------------------
+
+First, load the ``net.box`` module with:
+
+..  code-block:: tarantoolsession
+
+    tarantool> net_box = require('net.box')
+
+During the sandbox setup, ``box.cfg{...listen="3301"}`` was called.
+It means that the local server listen address is ``3301``.
+If the connection fails, check the actual listen address.
+
+The next step is to create a new connection.
+In ``net.box``, self-connection is pre-established.
+That is, ``conn = net_box.connect('localhost:3301')`` can be replaced with the following command:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn = net_box.self
+
+Then, make a ping:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn:ping()
+    ---
+    - true
+    ...
+
+Using data operations
```

Comment:
Idea: add more headings for easy navigation to operations. 
Smth like:
Using data operations
  - Select
  - Insert
  - Read
  - ...

Some sections may be too short as I see, please check how this will look on the page.